core/src/main/java/hudson/Util.java

```diff
@@ -1290,26 +1289,16 @@ public static String resolveSymlink(@Nonnull File link) throws IOException {
      */
     static @Nonnull File createJunction(@Nonnull File junction, @Nonnull File target) throws IOException, InterruptedException {
         if (!Functions.isWindows()) throw new AssertionError("Cannot create junctions on non-Windows platforms");
-        String command = "mklink /J " + escapeCmdArgument(junction.getAbsolutePath()) + ' ' + escapeCmdArgument(target.getAbsolutePath());
-        Process process = new ProcessBuilder("cmd.exe", "/C", command).start();
+        String command = "cmd.exe /C mklink /J " +
+                WindowsCommandLineFormatter.quoteArgumentForCmd(junction.getAbsolutePath()) +
+                ' ' +
+                WindowsCommandLineFormatter.quoteArgumentForCmd(target.getAbsolutePath());
+        Process process = Runtime.getRuntime().exec(command);
```

Comment:
would this have been easier if you used the array version rather than the single string version?